packages/icon/src/Icon.spec.tsx

```diff
@@ -1,20 +1,71 @@
 import React from 'react';
+import path from 'path';
+import fs from 'fs';
+import { toJson } from 'xml2json';
 import { render, cleanup } from '@testing-library/react';
 import { typeIs } from '@leafygreen-ui/lib';
 import { createIconComponent } from '.';
 
 afterAll(cleanup);
 
+describe('packages/Icon/glyphs/', () => {
+  const glyphPaths = fs
+    .readdirSync(path.resolve(__dirname, './glyphs'))
+    /**
+     * './glyphs/' contains an index.ts file, so we filter out
```

Comment:
Can we also match that this list aligns with the exported object given from `index.ts`?

Reply:
Done!